src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,161 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import {
+  FacebookSvg,
+  InstagramSvg,
+  TwitterSvg,
+  XingSvg,
+  YoutubeSvg,
+  LinkedinSvg,
+  CaretSvg,
+} from '@axa-ch/materials';
+import '@axa-ch/container';
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import { repeat } from 'lit-html/directives/repeat';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXAFooter extends LitElement {
+  static get tagName() {
+    return 'axa-footer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      content: { type: Array },
+      social: { type: Object },
+      dynamic: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.content = [];
+    this.social = {};
+    this.dynamic = false;
+    this.onItemClick = () => {};
+
+    this.iconMap = new Map();
+    this.iconMap.set('facebook', FacebookSvg);
+    this.iconMap.set('instagram', InstagramSvg);
+    this.iconMap.set('twitter', TwitterSvg);
+    this.iconMap.set('xing', XingSvg);
+    this.iconMap.set('youtube', YoutubeSvg);
+    this.iconMap.set('linkedin', LinkedinSvg);
+  }
+
+  handleClick(e, index) {
+    e.stopPropagation();
+    const oldState = this.content[index].active;
+    this.content = this.content.map(c => {
+      c.active = false;
+      return c;
+    });
+
+    this.content[index].active = !oldState;
+  }
+
+  handleLinkClick = (ev, text) => {
+    if (this.dynamic) {
+      ev.preventDefault();
+      this.onItemClick(text);
+      this.dispatchEvent(
+        new CustomEvent('axa-link-click', {
+          detail: text,
+          bubbles: true,
+          cancelable: true,
+        })
+      );
+    }
+  };
+
+  render() {
+    return html`
+      <footer class="o-footer">
+        <axa-container>
+          <div class="o-footer__content">
+            <div class="o-footer__collection">
+              ${repeat(
+                this.content,
+                (contentItem, index) =>
+                  html`
+                    <div class="o-footer__main">
+                      <strong class="o-footer__title-desktop"
```

Comment:
This has to be a title `<h1>` or `<h2>` and therefore `<footer>` tag wont work anymore due to the `h1` tags. 

This is the outline of the footer ATM which is not possible to have for SEO:

<img width="617" alt="Bildschirmfoto 2019-05-28 um 09 26 02" src="https://user-images.githubusercontent.com/275235/58459061-d8f42800-812a-11e9-912d-9548e594d960.png">